website/users/index.html

```diff
@@ -4,288 +4,214 @@
-                It takes just a few seconds for a new gVisor container to
-                start up and begin executing meaningful work in a secure
-                sandbox with near native performance.
-                (<a href="https://blog.cloudflare.com/cloudflare-pages-build-improvements/">A
-                  new era for Cloudflare Pages builds</a>)
-              </p>
-          </div>
-        </div> <!-- end panel -->
-      </div>
-    </div>
-    <hr>
-    <!-- DigitalOcean -->
-    <div class="row display-flex no-space">
-      <div class="col-md-12">
-        <h2>
-          <a href="https://www.digitalocean.com/" class="feature-link">
-            DigitalOcean
-          </a>
-        </h2>
-        <p class="info-text"> DigitalOcean is a cloud computing provider that
-        offers cloud infrastructure services to developers and businesses.
-        </p>
-        <p class="info-users-text">DigitalOcean uses gVisor in
-          <a href="https://docs.digitalocean.com/products/app-platform/">App Platform</a>
-          as a container runtime sandbox
-          (<a href="https://docs.digitalocean.com/products/app-platform/details/limits/">source</a>).
-        </p>
-      </div>
-    </div>
-    <hr>
-    <!-- Docker -->
-    <div class="row display-flex no-space">
-      <div class="col-md-12">
-        <h2>
-          <a href="https://www.docker.com/" class="feature-link">
-            Docker
-          </a>
-        </h2>
-        <p class="info-text">
-          Docker is a popular container management engine.
-        </p>
-        <p class="info-users-text">
-          Docker for Mac
-          <a href="https://docs.docker.com/desktop/release-notes/#4190">uses the gVisor network
-            stack library</a> for better performance than <code>vpnkit</code>.
-          Note that on Docker for Linux, you can
-          <a href="/docs/user_guide/quick_start/docker/">use gVisor as a container runtime</a>.
-        </p>
-      </div>
-    </div>
-    <hr>
-    <!-- Freedom of the Press Foundation -->
-    <div class="row display-flex no-space">
+    <p>This page lists companies known to use gVisor. This does not constitute an endorsement.</p>
+  </header>
+
+  <main class="users-list">
+    
+    <section class="user-company mb-5">
+      <h2>
+        <a href="https://www.antgroup.com/en" class="link-primary" target="_blank" rel="noopener noreferrer">
+          Ant Group
+        </a>
+      </h2>
+      <p>Ant Group develops online payment platforms, offering a wide range of financial services globally.</p>
+      <blockquote class="user-quote p-3 bg-light border rounded">
+        At Ant Group, we are committed to keeping online transactions safe and efficient. Continuously improving security for potential system-level attacks is one of many measures. As a container runtime, gVisor provides container-native security without sacrificing resource efficiency. It has been on our radar since its release.
```

Comment:
We try to wrap these lines at 80 chars. If you see the older version, all the <p> </p> content was wrapped. Please do the same here.